official/transformer/transformer_main.py

```diff
@@ -392,7 +397,7 @@ def define_transformer_flags():
       intra_op=False,
       synthetic_data=True,
       max_train_steps=False,
-      dtype=False,
+      dtype=True,
```

Comment:
If you set dtype=True, it also defines --loss_scale, which is ignored. I would add a loss_scale argument to define_performance, so you can pass False to it.

Reply:
Done and loss_scale is False by default and I updated resnet.